packages/generator-one-app-module/README.md

```diff
@@ -0,0 +1,21 @@
+# @americanexpress/generator-one-app-module
+
+> yeoman generator for a bare-bones [One App](https://github.com/americanexpress/one-app#modules) `holocron` Module
+
+## 🤹‍ Usage
+
+Assuming you have [npx installed](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b) (comes with `npm` on versions 5.2.0 and above):
+
+```bash
+npx -p yo -p @americanexpress/generator-one-app-module -- yo one-app-module
+```
+
+## 🏆 Contributing
+
+Please see our [contributing guide](../../CONTRIBUTING.md) for details.
```

Comment:
I think it is probably more intuitive to have this line after the making changes to the generator section.